docs/pages/ktooltip.vue

```diff
@@ -104,6 +105,30 @@
         <button ref="maxWidthButton">Learn More</button>
       </DocsShowCode>
       <p>The <code>maxWidth</code> prop can be used to limit the tooltip width.</p>
+
+      <h3>Appearance Overrides</h3>
```

Comment:
```suggestion
      <h3>Appearance overrides</h3>
```

As per writing guidelines :)